<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinkedListWidget</class>
 <widget class="QWidget" name="LinkedListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="pageTitle">
     <property name="text">
      <string>Create A Linked List!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item>
    <widget class="QLineEdit" name="nodeValueInput"/>
   </item>
   <item>
    <widget class="QPushButton" name="addNode">
     <property name="text">
      <string>Add Node</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="deleteNode">
     <property name="text">
      <string>Delete Node</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="searchForNode">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="sortButton">
     <property name="text">
      <string>Sort List</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="resetButton">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="nodeColorButton">
     <property name="text">
      <string>Node Color</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
